packages/eslint-plugin-ecmascript-compat/lib/polyfilling.js

```diff
@@ -0,0 +1,8 @@
+function nonPolyfilledFeatures(features, polyfills) {
```

Comment:
In another comment I proposed an `isPolyfilled()` function instead.

Like this one, it should be small enough that it can live in `rule.js`

Reply:
>  it should be small enough that it can live in rule.js

I'm future-proofing here as I thought about moving polyfill strings to constants (enum) that can be re-used when defining schema. I wanted to put them into `polyfilling.js` file.

Reply:
I see. There is this from before though:

> Schema should validate that only known polyfills are specified. The valid strings could be pulled off the rule definitions programmatically, but it's perhaps better just to hard code all the strings here so it can serve as documentation.

By documentation, I mean for users using the plugin, rather than for developers of this plugin. I don't think they should have to follow constants through the codebase to find out the accepted polyfills - they should just be able to look at the schema. The draft schema you posted earlier looks good for that.

Reply:
Ok. Makes sense to me. Let's remove polyfiling.js file. It's simple call and truth to be told it doesn't really deserve to be a separate func call on its own.